tests/filecheck/dialects/stencil/stencil_ops.mlir

```diff
@@ -202,3 +202,39 @@ builtin.module {
 // CHECK-NEXT:     func.return
 // CHECK-NEXT:   }
 // CHECK-NEXT: }
+
+// -----
+
+builtin.module {
+  func.func private @stencil_dyn_access(%0 : !stencil.field<?x?xf64>, %1 : !stencil.field<?x?xf64>) {
+    %2 = "stencil.cast"(%0) : (!stencil.field<?x?xf64>) -> !stencil.field<[-4,68]x[-4,68]xf64>
+    %3 = "stencil.cast"(%1) : (!stencil.field<?x?xf64>) -> !stencil.field<[-4,68]x[-4,68]xf64>
+    %4 = "stencil.load"(%2) : (!stencil.field<[-4,68]x[-4,68]xf64>) -> !stencil.temp<[-1,65]x[-1,65]xf64>
+    %5 = "stencil.apply"(%4) ({
+    ^0(%6 : !stencil.temp<[-1,65]x[-1,65]xf64>):
+      %i = "stencil.index"() {"dim" = 0 : i64, "offset" = #stencil.index<-1, -1>} : () -> index
+      %j = "stencil.index"() {"dim" = 1 : i64, "offset" = #stencil.index<1, 1>} : () -> index
+      %7 = "stencil.dyn_access"(%6, %i, %j) {"lb" = #stencil.index<-1, -1>, "ub" = #stencil.index<1, 1>} : (!stencil.temp<[-1,65]x[-1,65]xf64>, index, index) -> f64
+      "stencil.return"(%7) : (f64) -> ()
+    }) : (!stencil.temp<[-1,65]x[-1,65]xf64>) -> !stencil.temp<[0,64]x[0,64]xf64>
+    "stencil.store"(%5, %3) {"lb" = #stencil.index<0, 0>, "ub" = #stencil.index<64, 64>} : (!stencil.temp<[0,64]x[0,64]xf64>, !stencil.field<[-4,68]x[-4,68]xf64>) -> ()
+    func.return
+  }
+}
+
```

Comment:
Same as input? SHould we see any difference?

Reply:
It's as expected, it's the simple roundtrip filecheck 🙂
Usefulness is debatable here but it's correct ahah